src/Raven.Server/Web/System/OngoingTasksHandler.cs

```diff
@@ -574,6 +574,21 @@ private void WriteResult(JsonOperationContext context, DynamicJsonValue dynamicJ
             }
         }
 
+
+        [RavenAction("/databases/*/subscription-tasks", "DELETE", AuthorizationStatus.ValidUser)]
+        public async Task DeleteSubscriptionTask()
+        {
+            // Note: Only Subscription task needs User authentication, All other tasks need Admin authentication
+            var typeStr = GetQueryStringValueAndAssertIfSingleAndNotEmpty("type");
+            if (Enum.TryParse<OngoingTaskType>(typeStr, true, out var type) == false)
+                throw new ArgumentException($"Unknown task type: {type}", nameof(type));
+
+            if (type != OngoingTaskType.Subscription)
```

Comment:
what if you pass different task type? Then you can remove subscription w/o access? 
I think we should check here if task with given type is actually subscription.

Reply:
If you pass anything else it will throw, which is what we want, no?

Reply:
what if some ETL has task id:5 and I say:
DELETE /databases/*/subscription-tasks?type=Subscription&taskId=5

Will it delete etl task?

Reply:
ok I can see we get task type again in: DeleteOngoingTask method, so it shouldn't be issue. Please ignore my comments then.